src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers, getTotalPagesNumber } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange:
+  (event: React.MouseEvent<HTMLAnchorElement>) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPagesNumber = getTotalPagesNumber(total, perPage);
+  const pagesToRender = getNumbers(1, totalPagesNumber);
+  const isFirstPageActive = currentPage === 1;
+  const isLastPageActive = currentPage === totalPagesNumber;
+
+  return (
+    <>
```

Comment:
Do you really need to wrap everything with a fragment?

Reply:
I don't. These were the remnants of my previous version of the Component